chapter2/tests/t5/chapter2-l2-t5-get-file-unique-extensions.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Task 5 - get all unique extensions of files
+set -eo pipefail
+
+
+# Error handling function.
+err() {
```

Comment:
Please make `err` function in the same way I asked you in previous PRs, i.e. add `[ERROR]` prefix and make `exit` from it